project/UpdateEclipseClasspaths.scala

```diff
@@ -32,15 +32,17 @@ object UpdateEclipseClasspaths extends App {
 
   def main(): Int = {
 
+    System.err.println("Updating Eclipse Classpaths")
     val epl = new java.io.File(dafHome).listFiles()
     val classPathFiles = epl.flatMap { f =>
       if (f.isDirectory())
         f.listFiles().filter { _.getName == ".classpath" }
       else Nil
     }.filterNot { _.toString.contains("daffodil-macro-lib") }
 
+    assert(!classPathFiles.isEmpty, "No subproject directories found. Check DAFFODIL_HOME environment variable.")
     classPathFiles.foreach { f =>
-      println("updating " + f.toString)
+      println("Updating " + f.toString)
```

Comment:
SBT standards are actually to start message with lowercase, so I think "updating" is correct if were following that.